packages/astro/src/core/routing/manifest/create.ts

```diff
@@ -451,6 +451,19 @@ export function createRouteManifest(
 			.map(([{ dynamic, content }]) => (dynamic ? `[${content}]` : content))
 			.join('/')}`.toLowerCase();
 
+		{
+			let destination: string
+			if (typeof to === "string") {
+				destination = to
+			}
+			else {
+				destination = to.destination
+			}
+			if (destination.startsWith("http")) {
```

Comment:
Is 'httpfoobar' a valid pathname destination? Do we enforce that the pathname starts with `/`? I guess not since this gets here. So should we be checking for `http(s)://`?

Reply:
Makes sense, we don't validate the paths much.